client/src/plus/app.tsx

```diff
@@ -1,56 +1,65 @@
-import React, { useEffect } from "react";
-import { useLocale } from "./../hooks";
+import React from "react";
+import Modal from "react-modal";
+import { useLocale } from "../hooks";
 
 import "./fonts/metropolis.css";
 import "./fonts/inter.css";
 import "./index.scss";
 
+Modal.setAppElement("main");
+
+function BackToMDNModal({ onRequestClose }) {
+  const locale = useLocale();
+
+  return (
+    <Modal
+      overlayClassName="modal"
+      className="modal-inner"
+      aria-expanded
+      onRequestClose={onRequestClose}
+      isOpen
+      parentSelector={() => document.querySelector(".plus main")!}
+    >
+      <div className="modal-inner">
+        <div className="modal-content">
+          <figure className="dino" />
+          <h1>Thank you for your interest in MDN Plus</h1>
+          <p>
+            We are grateful for all the feedback you sent our way. The waitlist
+            signup and survey are now closed, and we’re beginning work towards
```

Comment:
I never know what's best/prettiest but if we're going to use `’` instead of `'` we should use it line 32 too.